### DISCUSSION TITLE
Unable to Set Up Amazon SES for Sending Email from Mailu [SMTP]

### DISCUSSION THREAD

**Discussion by wilsonaissel** [Mailu/Mailu#3146] [User support]
I am currently facing difficulties in configuring Amazon SES to send emails through Mailu using SMTP. Despite following the documentation and relevant guidelines, I am encountering issues that prevent successful setup.
have referred to the official documentation of both Mailu and Amazon SES, ensuring that I follow the recommended steps for integration. However, despite my efforts, I have not been able to establish a functional connection between Mailu and Amazon SES for sending emails
Mailu version 2.0.16

**Comment by csthiang** [comment]
What is the error you are getting?

1. You need to make sure you have the domain that you want to send email verified with Amazon SES. Go to `Amazon SES > Configuration: Identities > Create Identity`
2. Create a SMTP credential by going to `Amazon SES > SMTP settings > Create SMTP Credentials`
3. You need to config the following in your environment variables with the credentials in step 2:

- RELAYHOST
  - e.g. `email-smtp.<replace_with_ses_region>.amazonaws.com:<port>`
- RELAYPASSWORD
  - e.g. password you get in step 2.
- RELAYUSER
  - e.g. username you get in step 2.
